src/pieces/commandManager.ts

```diff
@@ -14,6 +14,13 @@ async function register(bot: Client): Promise<void> {
 		bot.emit('error', error);
 	}
 
+	bot.on('messageCreate', async msg => {
+		if (msg.content.substring(0, 8).includes('s;enroll')) {
```

Comment:
You should check the lower cased version of the content since that is what the old command manager did.

Reply:
That's a very good point actually, someone could use `S;enroll` due to being on mobile and this if statement would return false